docs/examples/ConnectionlessOffer.md

```diff
@@ -1,28 +1,153 @@
+        },
+        "metaRetries": 5
+      }
+      ```
+
+3. The `Admin` forwards the `invitationUrl` on to the `Peer`
+
+4. The `Peer` checks the validity of the `invitationUrl` using `agent.parseInvitation`
+
+      ```typescript
+      const oob = await agent.parseInvitation(invitationUrl)
+      ```
+
+5. The `Peer` accepts the Invitation using `agent.acceptInvitation`
+
+      ```typescript
+      await agent.acceptInvitation(oob)
+      ```
+
+6. The `Issuer` will then send back the credential offer.
+7. The `Peer` listens for the credential offer response from the `Issuer`, and accepts it
+
+    ```typescript
+    agent.addListener(SDK.ListenerKey.MESSAGE, async (newMessages: SDK.Domain.Message[]) => {
+      // newMessages can contain any didcomm message that is received, including
+      // Credential Offers, Issued credentials and Request Presentation Messages
+      const credentialOffers = newMessages.filter(
+        (message) => message.piuri === 'https://didcomm.org/issue-credential/3.0/offer-credential'
```

Comment:
Proposing to change 'https://didcomm.org/issue-credential/3.0/offer-credential' to SDK.ProtocolType.DidcommOfferCredential